src/components/LayerUI.tsx

```diff
@@ -526,7 +462,7 @@ const LayerUI = ({
               actionManager={actionManager}
               showExitZenModeBtn={showExitZenModeBtn}
             >
-              {childrenComponents.FooterCenter}
+              {childrenComponents.FooterCenter?.props.children}
```

Comment:
didn't notice last time, but why do we do this? Can't we just pass the `FooterCenter` component as prop and render it as is inside the `<Footer>`?

Reply:
no this was the bug in prod I was talking about as the footer renders twice so I just pass the children of the footer
<img width="509" alt="Screenshot 2023-01-05 at 12 52 20 PM" src="https://user-images.githubusercontent.com/11256141/210724277-fb2c85f0-1a11-48cf-83d9-6b31aa6a35a0.png">

Reply:
[`9d60bca` (#6034)](https://github.com/excalidraw/excalidraw/pull/6034/commits/9d60bca8f1b01331231745a5dc9c2c35bdd867c9)

Called the commit "simplify footer rendering", but it's not that much about simplification as about making it clearer. Especially as we need to pass more things down (next PR). And to not render custom footer when none is passed from host.